abm/tests/test_exploration_strategy.py

```diff
@@ -0,0 +1,103 @@
+import unittest
+import numpy as np
+from abm.exploration_strategy import (
+    ExplorationStrategy,
+    RandomWalkerExplorationStrategy,
+)
+
+
+class TestExplorationStrategy(unittest.TestCase):
```

Comment:
Please change the testing framework to `pytest`

Reply:
shifted to pytest